cmd/pebble/cmd_logs.go

```diff
@@ -15,27 +15,129 @@
 package main
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"strconv"
+	"time"
+
 	"github.com/jessevdk/go-flags"
+
+	"github.com/canonical/pebble/client"
 )
 
 type cmdLogs struct {
 	clientMixin
+	Follow     bool   `short:"f" long:"follow"`
+	Format     string `long:"format"`
```

Comment:
journalctl uses `-o` and `--output` for format selection.
kubectl uses `-o` and `--output` for format selection in `kubectl get`.

I think it makes more sense as -o can be the flag on commands other than logs as well.

Reply:
Fair enough. `--format` is also a bit confusing with `-f` meaning something different. (OTOH, Juju uses `--format`, and `-o` means "write to named output file" in many tools, like `wget` and `curl`).

Reply:
Done.